public/app/features/alerting/unified/Receivers.tsx

```diff
@@ -143,6 +144,17 @@ const Receivers = () => {
           <Route exact={true} path="/alerting/notifications/templates/new">
             <NewTemplateView config={config} alertManagerSourceName={alertManagerSourceName} />
           </Route>
+          <Route exact={true} path="/alerting/notifications/templates/:name/duplicate">
+            {({ match }: RouteChildrenProps<{ name: string }>) =>
+              match?.params.name && (
```

Comment:
I'm not exactly sure why we need to check for the name param here (since we're using `exact`) but I think this was just copied from the edit route.

Reply:
yes, that was copied from the edit route 😁, let me check it

Reply:
probably we need it to pass the `templateName` as a decodedURI?

`templateName={decodeURIComponent(match?.params.name)`